packages/rollup-plugin-import-meta-assets/src/rollup-plugin-import-meta-assets.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const { createFilter } = require('@rollup/pluginutils');
+const { asyncWalk } = require('estree-walker');
+const MagicString = require('magic-string');
+
+function getRelativeAssetPath(node) {
+  return node.arguments[0].value;
```

Comment:
This will return a browser path while it's being used as a file system path later. That won't work on windows.

You can transform it like this: `browserPath.split('/').join(path.sep)`